Library/Homebrew/rubocops/cask/desc.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "forwardable"
+require "uri"
+require "rubocops/cask/mixin/on_desc_stanza"
+require "rubocops/shared/desc_helper"
+
+module RuboCop
+  module Cop
+    module Cask
+      # This cop checks that a cask's homepage ends with a slash
+      # if it does not have a path component.
```

Comment:
You forgot to update this comment :)

Reply:
Maybe just copy the comment from the formula cop.

Reply:
Of course, makes sense. Done.